### DISCUSSION TITLE
About vips_init and vips_shutdown question

### DISCUSSION THREAD

**Discussion by zhaohuxing** [libvips/libvips#2540] [Q&A] [answered]
Hi @jcupitt 

If I call vips_init, but do not call vips_shutdown, will there be any consequences？

**Comment by jcupitt** [comment, marked as answer]
Hi @zhaohuxing,

The libvips leak checker won't work (you'll see lots of warnings if you run with `VIPS_LEAK` set), but that's all.

**Reply by zhaohuxing** [reply]
Thanks

**Reply by jcupitt** [reply]
I expanded the vips_shutdown docs a bit:

https://github.com/libvips/libvips/commit/80f2581e0cf4047a0beafbca4d4600468d69c8aa
